jobs/JOBSPROC_MODS

```diff
@@ -126,30 +134,31 @@ export SENDECF=${SENDECF:-YES}
 # Paths for obsproc items
 # -------------------------------
 export HOMEobsproc=${HOMEobsproc:\
--${NWROOT:?}/obsproc.${obsproc_ver:?\
+-${PACKAGEROOT:?}/obsproc/${obsproc_ver:?\
 "###FATAL ERROR \"obsproc_ver\" is not set"}}
 
 # ---------------------------------------
 # Paths for dump and bufr_dumplist items
 # ---------------------------------------
-export HOMEbufr_util=${HOMEbufr_util:\
--${NWROOT:?}/bufr_util.${bufr_util_ver:?\
-"###FATAL ERROR \"bufr_util_ver\" is not set"}}
+export HOMEbufr_dump=${HOMEbufr_dump:\
+-${PACKAGEROOT:?}/bufr_dump.${bufr_dump_ver:?\
+"###FATAL ERROR \"bufr_dump_ver\" is not set"}}
 
-export USHbufr_util=${USHbufr_util:-$HOMEbufr_util/ush}
-export FIXbufr_util=${FIXbufr_util:-$HOMEbufr_util/fix}
+export USHbufr_dump=${USHbufr_dump:-$HOMEbufr_dump/ush}
+export FIXbufr_dump=${FIXbufr_dump:-$HOMEbufr_dump/fix}
 
-# NOTE: path to dump executables defaults to $HOMEbufr_util/exec in dumpjb
+# NOTE: path to dump executables defaults to $HOMEbufr_dump/exec in dumpjb
 
 # directory path to data dump script executed in model script
 # -----------------------------------------------------------
-export ushscript_dump=${ushscript_dump:-$USHbufr_util}
+export ushscript_dump=${ushscript_dump:-$USHbufr_dump}
 
 export EXECmods=${EXECmods:-$HOMEobsproc/exec}
 export USHmods=${USHmods:-$HOMEobsproc/ush}
 export BTABdir=${BTABDIR:-$HOMEobsproc/fix}
 
-export DUMP=${DUMP:-$HOMEbufr_util/ush/dumpjb}
+export DUMP=$DUMPJB
+#export DUMP=${DUMP:-$HOMEbufr_dump/ush/dumpjb}
```

Comment:
I suggest consolidating and turning into this line instead: 
export DUMP==${DUMP:-$DUMPJB}

Reply:
I made a typo ... 
export DUMP=${DUMP:-$DUMPJB}